### DISCUSSION TITLE
How to Open SFTP

### DISCUSSION THREAD

**Discussion by wwy451304438** [Eugeny/tabby#7080] [Q&A]
Do you know how to display the SFTP toolbar after hiding it

**Comment by Eugeny** [comment]
Hover oder the little wrench icon where the unpin button used to be (top right)

**Reply by wwy451304438** [reply]
The mouse hover is invalid, and the small wrench is not found, perhaps no solution

**Reply by xueBengK** [reply]
brother, you need to save your ssh connect in tabby's configure and connecting, and connect to your server by using the connecting which you have saved. You will find the wrench icon. Don't use ssh directly! If you use ssh in Windows Powershell to connect your server then you can't find wrench icon. Hoping my answer will help you~ I meet the same problem and slove now!
